include/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth.hpp

```diff
@@ -49,16 +49,34 @@ class Earth : public Model
     {
         Undefined,    ///< Undefined
         Exponential,  ///< Exponential atmospheric density model, valid up to 1000 km
-        NRLMSISE00    ///< Navy Research Lab Mass Spectrometer and Incoherent Scatter Radar Exosphere 2000
+        NRLMSISE00,   ///< Navy Research Lab Mass Spectrometer and Incoherent Scatter Radar Exosphere 2000
+    };
+
+    enum class InputDataSourceType
+    {
+        Undefined,              ///< Undefined
+        ConstantFluxAndGeoMag,  ///< Use constant values for F10.7, F10.7a and Kp NRLMSISE00 input parameters
+        SpaceWeatherFile,  ///< Use historical and predicted values for F10.7, F10.7a and Kp NRLMSISE00 input parameters
```

Comment:
```suggestion
        CSSISpaceWeatherFile,  ///< Use historical and predicted values for F10.7, F10.7a and Kp NRLMSISE00 input parameters
```
nit: can we name this with `CSSI`? If we have a `Schatten` in the future, that will also presumably be a space-weather file. Also, CSSI is more searchable for someone trying to understand what's happening

Reply:
Yeah good point. I wanted to make that change when we would have added a new type like Schatten later but you're right: that will be yet another breaking change, better now 💯

Reply:
Done